Library/Formula/racket.rb

```diff
@@ -28,17 +31,43 @@ def install
       system "make"
       system "make", "install"
     end
+
+    # configure racket's package tool (raco) to do the Right Thing
+    # see: http://docs.racket-lang.org/raco/config-file.html
+    inreplace etc/"racket/config.rktd" do |s|
+        s.gsub!(
+            /\(bin-dir\s+\.\s+"#{Regexp.quote(bin)}"\)/,
+            "(bin-dir . \"#{HOMEBREW_PREFIX}/bin\")",
+        )
+        s.gsub!(
+            /\n\)$/,
+            "\n      (default-scope . \"installation\")\n)",
+        )
+    end
```

Comment:
Rereading: yes, I see, sorry (apologies, I'm going through a lot of PRs this evening). Could you append to the file instead of using `inreplace` here?